pkg/hintrunner/zero/zerohint_ec_test.go

```diff
@@ -593,5 +593,50 @@ func TestZeroHintEc(t *testing.T) {
 				}),
 			},
 		},
-	})
+		"ReduceV1" :{
+			{
+				operanders: []*hintOperander{
+					{Name: "x.d0", Kind: apRelative, Value: &utils.FeltZero},
+					{Name: "x.d1", Kind: apRelative, Value: &utils.FeltZero},
+					{Name: "x.d2", Kind: apRelative, Value: &utils.FeltZero},
+				},
+				ctxInit: func(ctx *hinter.HintRunnerContext) {
+					hinter.InitializeScopeManager(ctx, map[string]any{})
+				},
+				makeHinter: func(ctx *hintTestContext) hinter.Hinter {
+					return newReduceV1Hinter(ctx.operanders["x.d0"])
+				},
+				check: varValueEquals("x.d0", feltString("0")),
```

Comment:
you need to check if the value in scope is the expected one. You can use the function:
```
check: varValueInScopeEquals("x",bigIntString("put_the_value_here", 10))
```